src/vivarium_csu_ltbi/data/ltbi_incidence_model.py

```diff
@@ -1,28 +1,27 @@
 import pandas as pd
 import numpy as np
-import dismod_mr
```

Comment:
Hid these imports in the functions so users don't get unneccesary import errors.